docs/building.md

```diff
@@ -0,0 +1,25 @@
+# Building Parthenon on Various Systems
+
+## Ubuntu 20.04 LTS
+
+The following procedure has been tested for an Ubuntu 20.04 LTS system:
+
+```bash
+# install dependencies
+# openmpi is installed implicitly by the hdf5 install
+sudo apt-get update
+install cmake build-essentials libhdf5-openmpi-dev
+
+# Clone parthenon, with submodules
+git clone --recursive https://github.com/lanl/parthenon.git
+# make a bin directory
+mkdir bin
+cd bin
+# configure and build
+cmake ..
+make -j --build .
```

Comment:
`cmake` not `make`.

Reply:
Oops. not sure how I managed that. I'll fix it.

Reply:
Huh?

I really are two cmake call:
```
cmake ..
cmake --build .
```

Reply:
Oh I see. Sorry, I misunderstood.